apps/tutorial-servo-pot-control/build/.cmake/api/v1/query/client-vscode/query.json

```diff
@@ -0,0 +1 @@
+{"requests":[{"kind":"cache","version":2},{"kind":"codemodel","version":2}]}
```

Comment:
i dont think u meant to push this

Reply:
Yeah the build files came from me trying to build through vscode, totally forgot to delete them